knodle/labeler/CheXpert/label.py

```diff
@@ -0,0 +1,62 @@
+"""
+This code builds upon the CheXpert labeler from Stanford ML Group.
+It has been slightly modified to be compatible with knodle.
+The original code can be found here: https://github.com/stanfordmlgroup/chexpert-labeler
+
+----------------------------------------------------------------------------------------
+
+Entry-point script to label the provided reports.
+"""
+import os
+import shutil
+import numpy as np
+
+from .config import CheXpertConfig
+from .utils import t_matrix_fct
+from . import Preprocessor, Matcher, NegUncDetector, Updater
+from knodle.labeler.labeler import Labeler
+
+
+class CheXpertLabeler(Labeler):
+
+    def __init__(self, **kwargs):
+        if kwargs.get("labeler_config", None) is None:
+            kwargs["labeler_config"] = CheXpertConfig()
+        super().__init__(**kwargs)
```

Comment:
kwargs is a sensitive object, errors easily happen, I'd rather put it explicitly:
```
    def __init__(self, labeler_config=None, **kwargs):
        labeler_config = CheXpertConfig() if labeler_config is None else labeler_config
        super().__init__(labeler_config, **kwargs)
```